### ISSUE TITLE
No mention of Windows 11 on file extensions (windows)

### ISSUE THREAD

**Issue by lisanderke** [hacks-guide/Guide_3DS#1981]
**Pages with issue(s)**
file-extensions-(windows).txt
File Extensions (Windows)
https://3ds.hacks.guide/file-extensions-(windows)

**Description of the issue(s)**
There is no mention of the Windows 11 OS on this page. To improve accessibility for people with new laptops/desktop PCs (as these have Windows 11 preinstalled now), I think an additional section to this page would be useful. Though it should be noted that only fewer than 1% of Windows devices are running this specific OS, as such it's probably not that important of an issue right now.
It should also be noted that simply changing the title "Windows 10 / 8.1 / 8" to "Windows 11 / 10 / 8.1 / 8" won't suffice as Windows 11 changes the File Explorer UI and the steps mentioned on this page are incorrect for this specific OS.

**Comment by ihaveamac** [comment]
Windows 11 steps are added in a467da9d43fe0ad17e6937ca8b0283baf9b50010, someone else will need to provide a screenshot since I don't have a suitable PC for it.
